IT logs with error:
```
[INFO] Running com.google.cloud.vision.it.ITSystemTest
WebEntitiesGeo SIZE
0
[ERROR] Tests run: 33, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 20.672 s <<< FAILURE! - in com.google.cloud.vision.it.ITSystemTest
[ERROR] com.google.cloud.vision.it.ITSystemTest.detectWebEntitiesGcsTest  Time elapsed: 3.052 s  <<< FAILURE!
expected to contain: Palace Of Fine Arts
but was            : [Ancient Rome, Ancient history, Triumphal arch, Ancient Roman architecture, Palace of Fine Arts, History, Architecture, Historic site, Façade, Arch, Tourism, Hematocrit]
	at com.google.cloud.vision.it.ITSystemTest.detectWebEntitiesGcsTest(ITSystemTest.java:495)

[ERROR] com.google.cloud.vision.it.ITSystemTest.detectWebEntitiesIncludeGeoResultsGcsTest  Time elapsed: 1.607 s  <<< FAILURE!
expected to contain: Palace Of Fine Arts
but was            : [Ancient Rome, Triumphal arch, Ancient history, Ancient Roman architecture, Palace of Fine Arts, History, Historic site, Architecture, Façade, Arch, Tourism, Hematocrit]
	at com.google.cloud.vision.it.ITSystemTest.detectWebEntitiesIncludeGeoResultsGcsTest(ITSystemTest.java:557)
```

--- a/google-cloud-vision/src/test/java/com/google/cloud/vision/it/ITSystemTest.java
+++ b/google-cloud-vision/src/test/java/com/google/cloud/vision/it/ITSystemTest.java
@@ -473,7 +473,7 @@ public class ITSystemTest {
   }
 
   @Test
-  public void detectWebEntitiesGcsTest() throws IOException {
+  public void detectWebEntitiesGcsTest() {
     ImageSource imgSource =
         ImageSource.newBuilder().setGcsImageUri(SAMPLE_BUCKET + "landmark/pofa.jpg").build();
     Image img = Image.newBuilder().setSource(imgSource).build();
@@ -492,6 +492,7 @@ public class ITSystemTest {
         actual.add(entity.getDescription());
       }
     }
+    System.out.println(actual.contains("Palace of Fine Arts"));
     assertThat(actual).contains("Palace Of Fine Arts");
   }
 
@@ -547,13 +548,12 @@ public class ITSystemTest {
         imageAnnotatorClient.batchAnnotateImages(ImmutableList.of(request));
     List<AnnotateImageResponse> responses = response.getResponsesList();
     List<String> actual = new ArrayList<>();
-    System.out.println("WebEntitiesGeo SIZE");
-    System.out.println(actual.size());
     for (AnnotateImageResponse imgResponse : responses) {
       for (WebDetection.WebEntity entity : imgResponse.getWebDetection().getWebEntitiesList()) {
         actual.add(entity.getDescription());
       }
     }
+    System.out.println(actual.contains("Palace of Fine Arts"));
     assertThat(actual).contains("Palace Of Fine Arts");
   }
 

--- a/google-cloud-vision/src/test/java/com/google/cloud/vision/it/ITSystemTest.java
+++ b/google-cloud-vision/src/test/java/com/google/cloud/vision/it/ITSystemTest.java
@@ -493,7 +493,7 @@ public class ITSystemTest {
       }
     }
     System.out.println(actual.contains("Palace of Fine Arts"));
-    assertThat(actual).contains("Palace Of Fine Arts");
+    assertThat(actual).contains("Palace of Fine Arts");
   }
 
   @Test
@@ -554,7 +554,7 @@ public class ITSystemTest {
       }
     }
     System.out.println(actual.contains("Palace of Fine Arts"));
-    assertThat(actual).contains("Palace Of Fine Arts");
+    assertThat(actual).contains("Palace of Fine Arts");
   }
 
   @Test

--- a/google-cloud-vision/src/test/java/com/google/cloud/vision/it/ITSystemTest.java
+++ b/google-cloud-vision/src/test/java/com/google/cloud/vision/it/ITSystemTest.java
@@ -492,7 +492,6 @@ public class ITSystemTest {
         actual.add(entity.getDescription());
       }
     }
-    System.out.println(actual.contains("Palace of Fine Arts"));
     assertThat(actual).contains("Palace of Fine Arts");
   }
 
@@ -553,7 +552,6 @@ public class ITSystemTest {
         actual.add(entity.getDescription());
       }
     }
-    System.out.println(actual.contains("Palace of Fine Arts"));
     assertThat(actual).contains("Palace of Fine Arts");
   }
 

